mean-slopes/Lines_Slopes_and_Intercepts_-_Refresher.Rmd

```diff
@@ -0,0 +1,581 @@
+
+That's all there is to it. Adding or subtracting an intercept from a line simply moves the line up and down the y axis. If the rest of the equation is the same, that is all that changes. This is true regardless of what the slope is:
+
+```{python}
+# do not worry about any of the code in this notebook, it is just here to generate the graphs
+y1 = 10 * x
+y2 = 10 * x + 5
+plt.figure(figsize = (8,8))
+plt.plot(x,y1, color = 'blue', label = 'y = 10 * x')
+plt.plot(x,y2, color = 'gold', label = 'y = 10 * x + 5')
+y_axis = np.arange(-20, 21)
+plt.plot(x_axis, np.repeat(0, len(x_axis)), color = 'black')
+plt.plot(np.repeat(0, len(y_axis)), y_axis, color = 'black')
+plt.xlabel('X Axis')
+plt.ylabel('Y Axis')
+plt.plot(0,0, 'o', color = 'black', markersize = 8, label = 'The Origin (x = 0, y = 0)')
+plt.ylim(min(y_axis), max(y_axis))
+plt.legend()
+plt.show()
+```
+
+Let's see what happens if we set the intercept to -5, rather than postive 5:
+
+```{python}
+# do not worry about any of the code in this notebook, it is just here to generate the graphs
+y1 = 5 * x
+y2 = 5 *  x + 5
+y3 = 5 *  x - 5
+plt.figure(figsize = (8,8))
+plt.plot(x,y1, color = 'orange', label = 'y =  5 * x')
+plt.plot(x,y2, color = 'crimson', label = 'y = 5 * x + 5')
+plt.plot(x,y3, color = 'darkred', label = 'y = 5 * x - 5')
+plt.plot(x_axis, np.repeat(0, len(x_axis)), color = 'black')
+plt.plot(np.repeat(0, len(y_axis)), y_axis, color = 'black')
+plt.xlabel('X Axis')
+plt.ylabel('Y Axis')
+plt.plot(0,0, 'o', color = 'black', markersize = 8, label = 'The Origin (x = 0, y = 0)')
+plt.ylim(min(y_axis), max(y_axis))
+plt.legend()
+plt.show()
+```
+
+You can see that a negative intercept moves the line downwards away from the origin. 
+
+A positive intercept moves the line upwards away from the origin.
+
+This is also true if the slope is negative, as can be seen from the graph below:
+
+```{python}
+# do not worry about any of the code in this notebook, it is just here to generate the graphs
+y1 = -5 * x
+y2 = -5 *  x + 5
+y3= -5 *  x - 5
+plt.figure(figsize = (8,8))
+plt.plot(x,y1,'--', color = 'darkblue',  label = 'y =  -5 * x')
+plt.plot(x,y2,'--', color = 'silver',  label = 'y = -5 * x + 5')
+plt.plot(x,y3, '--',color = 'darkred',  label = 'y = -5 * x - 5')
+plt.plot(x_axis, np.repeat(0, len(x_axis)), color = 'black')
+plt.plot(np.repeat(0, len(y_axis)), y_axis, color = 'black')
+plt.xlabel('X Axis')
+plt.ylabel('Y Axis')
+plt.plot(0,0, 'o', color = 'black', markersize = 8, label = 'The Origin (x = 0, y = 0)')
+plt.ylim(min(y_axis), max(y_axis))
+plt.legend()
+plt.show()
+```
+
+### In summary: the slope determines the steepness of the line. The intercept moves the line up and down the y axis, relative to the origin.
+
+
+# Why do we care?
+
+Fitting a line to a set of datapoints can be used for description and prediction.
+
+The graph below shows some simulated data, where each point represents one person with a disease. 
```

Comment:
Can you find any real data you could use here?  It's always more interesting ...

Reply:
Hi Matthew, I've used the student ratings dataset instead of simulated data, please see 1075ab7